pallets/asset-manager/src/lib.rs

```diff
@@ -292,6 +326,12 @@ pub mod pallet {
 
         /// An error occurred while updating the parachain id.
         UpdateParaIdError,
+
+        /// Two asset that used for generate LP asset should different
+        AssetIdNotDifferent,
+
+        /// Two asset that used for generate LP asset should exist
+        AssetIdNotExist,
```

Comment:
is it still not possible to include inner types ?
Otherwise we can add the id and make it clearer.

Reply:
yes, not possible to include field in Error types.